test/test_functions_recreate.js

```diff
@@ -0,0 +1,58 @@
+exports.test = function(sql, assert) {
+  // Test 1: Create a database, Register single function, close database, repeat 1000 times
+  for (var i = 1; i <= 1000; i++) 
+  {
+    let db = new sql.Database();
+    function add() {return i;}
+    try
+    {
+      db.create_function("TestFunction"+i, add)
+    }catch(e)
+    {
+      assert.ok(false,"Test 1: Recreate database "+i+"th times and register function failed with exception:"+e);
```

Comment:
Can you please make a single `assert`, after the end of the loop ? Making 1000 asserts prints 1000 lines to the console.